.github/workflows/run-test.yml

```diff
@@ -0,0 +1,27 @@
+name: Run Tests
+
+on:
+  push:
+    branches: [ main, master ]
+  pull_request:
+    branches: [ main, master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    continue-on-error: ${{matrix.version == 'snapshot'}}
+
+    strategy:
+      matrix:
+        # Cask does't support Emacs 24.
+        version: ['25.1', '25.2', '25.3', '26.1', '26.2', '26.3', '27.1', '27.2', '28.1', '28.2', 'snapshot']
```

Comment:
does this work on all of these older versions of emacs? I believe I only ever really tested it as far back as 26 or 27